### DISCUSSION TITLE
How to remove input border

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [JedWatson/react-select#4963] [Q&A] [answered]
I have an input form, but it has a blue border, how do I remove it?
<img width="358" alt="Screen Shot 2021-12-21 at 14 45 36" src="https://user-images.githubusercontent.com/68141840/146891695-51c01a71-7b1f-4b6e-934a-ad9ecfd34681.png">

**Comment by Rall3n** [comment, marked as answer]
Hello @mhanifmuhsin,

this is a problem with whatever base styling framework you are using (e.g. TailwindCSS). Basically the framework applies styles to inputs with the following target: `input[type='text']:focused`. This can either be an `outline` or a `box-shadow`.

You can override it using the `Select` components Styling API:

```jsx
<Select
  styles={{
    input: (css) => ({
      ...css,
      input: {
        outline: none
      }
    })
  }}
/>
```
-----
Other solutions with `@tailwindcss/forms` as the cause of the issue can be found [here](https://github.com/JedWatson/react-select/issues/4686)

**Reply by mhanifmuhsin** [reply]
Thank's for your infromation
